workflow-ui/radiography/src/main/java/com/squareup/workflow1/ui/radiography/WorkflowViewRenderer.kt

```diff
@@ -0,0 +1,38 @@
+package com.squareup.workflow1.ui.radiography
+
+import com.squareup.workflow1.ui.Compatible
+import com.squareup.workflow1.ui.Named
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.getRendering
+import radiography.AttributeAppendable
+import radiography.ScannableView
+import radiography.ScannableView.AndroidView
+import radiography.ViewStateRenderer
+import radiography.ViewStateRenderers
+
+/**
+ * Renders information about views that were created by view factories, i.e. views with associated
+ * rendering tags.
+ */
+@Suppress("unused")
+public val ViewStateRenderers.WorkflowViewRenderer: ViewStateRenderer
```

Comment:
@pyricau WDYT about defining this as an extension on the object vs just a standalone object?